lib/editor/components/ExceptionDate.js

```diff
@@ -61,7 +67,7 @@ export default class ExceptionDate extends Component<Props, State> {
     if (dates.some(date => date === newDate && date !== 'Invalid date')) {
       dates.splice(index, 1)
       toast.warn(
-        `ⓘ Date has been removed. Date entered is already included in an existing range or single date!`,
+        this.messages('dateRemoved'),
```

Comment:
Does this case still apply?

Reply:
Yep! This is for within a single `ScheduleException`, where you've basically just added a duplicate date.